tools/fleet-slackbot/claude-client.js

```
@@ -108,9 +114,9 @@ class ClaudeClient {
 
       toolCallsExecuted += toExecute.length;
       messages.push({ role: "user", content: toolResults });
-      console.log(`[claude] Agent loop: ${toExecute.length} tool call(s) executed (${toolCallsExecuted}/${MAX_TOOL_CALLS}), ${toSkip.length} skipped`);
+      console.log(`[claude] Agent loop: ${toExecute.length} tool call(s) executed (${toolCallsExecuted}/${this.maxToolCalls}), ${toSkip.length} skipped`);
```

ESLint

[error] Replace ``[claude]·Agent·loop:·${toExecute.length}·tool·call(s)·executed·(${toolCallsExecuted}/${this.maxToolCalls}),·${toSkip.length}·skipped`` with `⏎········`[claude]·Agent·loop:·${toExecute.length}·tool·call(s)·executed·(${toolCallsExecuted}/${this.maxToolCalls}),·${toSkip.length}·skipped`⏎······`

(prettier/prettier)